tests/unit/s2n_map_test.c

```diff
@@ -133,6 +133,26 @@ int main(int argc, char **argv)
     key.size = strlen(keystr) + 1;
     EXPECT_EQUAL(s2n_map_lookup(map, &key, &val), 0);
 
+    /* Make the map mutable */
```

Comment:
should we first try to execute line 147 right here and verify that it fails? then succeeds after we call s2n_map_unlock?

Reply:
It is already covered in the existing tests, https://github.com/awslabs/s2n/blob/master/tests/unit/s2n_map_test.c#L98-L107